### ISSUE TITLE
Error when using with Vite

### ISSUE THREAD

**Issue by giotiskl** [porscheofficial/cookie-consent-banner#9]
**Node/npm version(s)**:
Node v18.7.0
npm v8.15.0

**Package(s) version(s)**:

```
 @porscheofficial/cookie-consent-banner-react@3.0.0
 @porscheofficial/cookie-consent-banner@3.0.0
```

**Browser(s) version(s)**:
Brave browser: Version 1.41.100 Chromium: 103.0.5060.134 (Official Build) (x86_64)

**Description of the issue/feature**:
I installed the package with yarn (v1.22.19). I imported it and tried to set up the basic example.
I get the following e
<img width="718" alt="Screenshot 2022-08-11 at 13 35 35" src="https://user-images.githubusercontent.com/12873414/184117104-23a5e8de-8f98-4d7b-9c5a-477cfbf191d9.png">
rrors in the browser console and banner is not rendered (the rest of the application compiles & renders):

**Some example to reproduce the issue**:
* Install porsche cookie consent banner in a basic Vite react project (bootstrapped with TS).
* Check the dev console.

**Comment by ppuritscher** [comment]
Hi @giotiskl! I'll look into this in a few days

**Comment by alvarosaburido-porsche** [comment]
Hi there, @giotiskl could you please provide a minimal reproduction using this https://stackblitz.com/edit/vitejs-vite-rqeyzy?file=index.html&terminal=dev ? 

That would help us a lot 

Thanks

**Comment by ppuritscher** [comment]
Does the latest version fix your issue @giotiskl (`v3.0.1`)?
